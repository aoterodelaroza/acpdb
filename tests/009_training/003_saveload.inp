## check: 003_saveload.out -a1e-10
## delete: 003_saveload.db
## labels: regression quick

system rm -f 003_saveload.db
connect 003_saveload.db

insert method test
end
insert method test2
end
insert method test3
end
insert litref bibtex ../zz_source/bsip2.bib
insert set 003_saveload-set-a
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22partial.din
 directory ../zz_source/s22_xyz
 method test
end
insert set 003_saveload-set-b
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method test
end
insert property strange-
 property_type dipole
 set 003_saveload-set-a
end
insert evaluation
 method test
 property strange-003_saveload-set-a.h2o_h2o
 value 0.1 0.2 0.3
end

training
 atom H l C l N s O p
 exp 0.1 0.2 0.3
 empty test2
 reference test
 add test3
 subset
  set 003_saveload-set-a
 end
 subset
  set 003_saveload-set-b
 end
end

training save bleh
training print
training clear
training describe
training load bleh
training describe
training delete bleh
training print
