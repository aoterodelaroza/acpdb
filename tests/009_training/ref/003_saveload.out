%% verbose
%% system rm -f 003_saveload.db
* SYSTEM: rm -f 003_saveload.db

%% connect 003_saveload.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 003_saveload.db
Creating skeleton database 

%% insert method test
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test

%% insert method test2
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test2

%% insert method test3
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test3

%% insert litref bibtex ../zz_source/bsip2.bib
* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

%% insert set 003_saveload-set-a
* INSERT: insert data into the database (SET)
# INSERT SET 003_saveload-set-a

%% insert set 003_saveload-set-b
* INSERT: insert data into the database (SET)
# INSERT SET 003_saveload-set-b

%% insert property strange-
* INSERT: insert data into the database (PROPERTY)
# INSERT PROPERTY strange-003_saveload-set-a.nh3_nh3
# INSERT PROPERTY strange-003_saveload-set-a.nh3_nh3_1
# INSERT PROPERTY strange-003_saveload-set-a.nh3_nh3_2
# INSERT PROPERTY strange-003_saveload-set-a.h2o_h2o
# INSERT PROPERTY strange-003_saveload-set-a.h2o_h2o_1
# INSERT PROPERTY strange-003_saveload-set-a.h2o_h2o_2

%% insert evaluation
* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=test;property=strange-003_saveload-set-a.h2o_h2o)

%% training
* TRAINING: started defining the training set 

%% atom H l C l N s O p
%% exp 0.1 0.2 0.3
%% empty test2
%% reference test
%% add test3
%% subset
%% subset
%% end
* TRAINING: fininshed defining the training set 

## Description of the training set
# List of atoms and maximum angular momentum channels (4)
| Atom | lmax |
| H | l |
| C | l |
| N | s |
| O | p |

# List of exponents (3)
| id | exp |
| 0 | 0.1 |
| 1 | 0.2 |
| 2 | 0.3 |

# List of subsets (2)
| id | alias | db-name | db-id | initial | final | size | dofit? | litref | description |
| 0 | 003_saveload-set-a | 003_saveload-set-a | 1 | 1 | 8 | 8 | 1 | bsip1 boys1970  | A description. |
| 1 | 003_saveload-set-b | 003_saveload-set-b | 2 | 9 | 30 | 22 | 1 | bsip1 boys1970  | A description. |

# List of methods
| type | name | id | for fit? |
| reference | test | 1 | n/a |
| empty | test2 | 2 | n/a |
| additional | test3 | 3 | no |

# List of properties (30)
| fit? | id | property | propid | alias | db-set | proptype | nstruct | weight | refvalue |
| yes | 1 | 003_saveload-set-a.nh3_nh3 | 1 | 003_saveload-set-a | 003_saveload-set-a | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 2 | strange-003_saveload-set-a.nh3_nh3 | 25 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 3 | 003_saveload-set-a.h2o_h2o | 2 | 003_saveload-set-a | 003_saveload-set-a | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |
| yes | 4 | strange-003_saveload-set-a.nh3_nh3_1 | 26 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 5 | strange-003_saveload-set-a.nh3_nh3_2 | 27 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 6 | strange-003_saveload-set-a.h2o_h2o | 28 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | <3> |
| yes | 7 | strange-003_saveload-set-a.h2o_h2o_1 | 29 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 8 | strange-003_saveload-set-a.h2o_h2o_2 | 30 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 9 | 003_saveload-set-b.nh3_nh3 | 3 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 10 | 003_saveload-set-b.h2o_h2o | 4 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |
| yes | 11 | 003_saveload-set-b.h2co2_h2co2 | 5 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -18.753000 |
| yes | 12 | 003_saveload-set-b.formamide_formamide | 6 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -16.062000 |
| yes | 13 | 003_saveload-set-b.uracil_uracil_hb | 7 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -20.641000 |
| yes | 14 | 003_saveload-set-b.pyridoxine_aminopyridine | 8 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -16.934000 |
| yes | 15 | 003_saveload-set-b.adenine_thymine_wcc1 | 9 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -16.660000 |
| yes | 16 | 003_saveload-set-b.ch4_ch4 | 10 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -0.527000 |
| yes | 17 | 003_saveload-set-b.c2h4_c2h4 | 11 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -1.472000 |
| yes | 18 | 003_saveload-set-b.c6h6_ch4 | 12 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -1.448000 |
| yes | 19 | 003_saveload-set-b.c6h6_c6h6_pd | 13 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -2.654000 |
| yes | 20 | 003_saveload-set-b.pyrazine_pyrazine | 14 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.255000 |
| yes | 21 | 003_saveload-set-b.uracil_uracil_stack | 15 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -9.805000 |
| yes | 22 | 003_saveload-set-b.indole_c6h6_stack | 16 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.524000 |
| yes | 23 | 003_saveload-set-b.adenine_thymine_stack | 17 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -11.730000 |
| yes | 24 | 003_saveload-set-b.c2h4_c2h2 | 18 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -1.496000 |
| yes | 25 | 003_saveload-set-b.c6h6_h2o | 19 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -3.275000 |
| yes | 26 | 003_saveload-set-b.c6h6_nh3 | 20 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -2.312000 |
| yes | 27 | 003_saveload-set-b.c6h6_hcn | 21 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.541000 |
| yes | 28 | 003_saveload-set-b.c6h6_c6h6_t | 22 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -2.717000 |
| yes | 29 | 003_saveload-set-b.indole_c6h6_t | 23 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -5.627000 |
| yes | 30 | 003_saveload-set-b.phenol_phenol | 24 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -7.097000 |

# Calculation completion for the current training set
# Reference: 25/30 (missing)
# Empty: 0/30 (missing)
# Additional (test3): 0/30 (missing)
# Terms: 
| H | l | 0.1 | 0/30 | (missing)
| H | l | 0.2 | 0/30 | (missing)
| H | l | 0.3 | 0/30 | (missing)
| C | l | 0.1 | 0/30 | (missing)
| C | l | 0.2 | 0/30 | (missing)
| C | l | 0.3 | 0/30 | (missing)
| N | l | 0.1 | 0/30 | (missing)
| N | l | 0.2 | 0/30 | (missing)
| N | l | 0.3 | 0/30 | (missing)
| N | s | 0.1 | 0/30 | (missing)
| N | s | 0.2 | 0/30 | (missing)
| N | s | 0.3 | 0/30 | (missing)
| O | l | 0.1 | 0/30 | (missing)
| O | l | 0.2 | 0/30 | (missing)
| O | l | 0.3 | 0/30 | (missing)
| O | s | 0.1 | 0/30 | (missing)
| O | s | 0.2 | 0/30 | (missing)
| O | s | 0.3 | 0/30 | (missing)
| O | p | 0.1 | 0/30 | (missing)
| O | p | 0.2 | 0/30 | (missing)
| O | p | 0.3 | 0/30 | (missing)
# Total terms: 0/630 (missing)

%% training save bleh
%% training print
## Table of saved training sets in the database
| Name |
| bleh |

%% training clear
%% training describe
## Description of the training set
# The TRAINING SET is NOT DEFINED
--- No atoms found (ATOM) ---
--- No angular momenta found (LMAX) ---
--- No exponents found (EXP) ---
--- No subsets found (SUBSET) ---
--- No weights found (W) ---
--- No empty method found (EMPTY) ---
--- No reference method found (REFERENCE) ---

%% training load bleh
%% training describe
## Description of the training set
# List of atoms and maximum angular momentum channels (4)
| Atom | lmax |
| H | l |
| C | l |
| N | s |
| O | p |

# List of exponents (3)
| id | exp |
| 0 | 0.1 |
| 1 | 0.2 |
| 2 | 0.3 |

# List of subsets (2)
| id | alias | db-name | db-id | initial | final | size | dofit? | litref | description |
| 0 | 003_saveload-set-a | 003_saveload-set-a | 1 | 1 | 8 | 8 | 1 | bsip1 boys1970  | A description. |
| 1 | 003_saveload-set-b | 003_saveload-set-b | 2 | 9 | 30 | 22 | 1 | bsip1 boys1970  | A description. |

# List of methods
| type | name | id | for fit? |
| reference | test | 1 | n/a |
| empty | test2 | 2 | n/a |
| additional | test3 | 3 | no |

# List of properties (30)
| fit? | id | property | propid | alias | db-set | proptype | nstruct | weight | refvalue |
| yes | 1 | 003_saveload-set-a.nh3_nh3 | 1 | 003_saveload-set-a | 003_saveload-set-a | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 2 | strange-003_saveload-set-a.nh3_nh3 | 25 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 3 | 003_saveload-set-a.h2o_h2o | 2 | 003_saveload-set-a | 003_saveload-set-a | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |
| yes | 4 | strange-003_saveload-set-a.nh3_nh3_1 | 26 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 5 | strange-003_saveload-set-a.nh3_nh3_2 | 27 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 6 | strange-003_saveload-set-a.h2o_h2o | 28 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | <3> |
| yes | 7 | strange-003_saveload-set-a.h2o_h2o_1 | 29 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 8 | strange-003_saveload-set-a.h2o_h2o_2 | 30 | 003_saveload-set-a | 003_saveload-set-a | DIPOLE | 1 | 1 | n/a |
| yes | 9 | 003_saveload-set-b.nh3_nh3 | 3 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -3.133000 |
| yes | 10 | 003_saveload-set-b.h2o_h2o | 4 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.989000 |
| yes | 11 | 003_saveload-set-b.h2co2_h2co2 | 5 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -18.753000 |
| yes | 12 | 003_saveload-set-b.formamide_formamide | 6 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -16.062000 |
| yes | 13 | 003_saveload-set-b.uracil_uracil_hb | 7 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -20.641000 |
| yes | 14 | 003_saveload-set-b.pyridoxine_aminopyridine | 8 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -16.934000 |
| yes | 15 | 003_saveload-set-b.adenine_thymine_wcc1 | 9 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -16.660000 |
| yes | 16 | 003_saveload-set-b.ch4_ch4 | 10 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -0.527000 |
| yes | 17 | 003_saveload-set-b.c2h4_c2h4 | 11 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -1.472000 |
| yes | 18 | 003_saveload-set-b.c6h6_ch4 | 12 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -1.448000 |
| yes | 19 | 003_saveload-set-b.c6h6_c6h6_pd | 13 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -2.654000 |
| yes | 20 | 003_saveload-set-b.pyrazine_pyrazine | 14 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.255000 |
| yes | 21 | 003_saveload-set-b.uracil_uracil_stack | 15 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -9.805000 |
| yes | 22 | 003_saveload-set-b.indole_c6h6_stack | 16 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.524000 |
| yes | 23 | 003_saveload-set-b.adenine_thymine_stack | 17 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -11.730000 |
| yes | 24 | 003_saveload-set-b.c2h4_c2h2 | 18 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -1.496000 |
| yes | 25 | 003_saveload-set-b.c6h6_h2o | 19 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -3.275000 |
| yes | 26 | 003_saveload-set-b.c6h6_nh3 | 20 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -2.312000 |
| yes | 27 | 003_saveload-set-b.c6h6_hcn | 21 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -4.541000 |
| yes | 28 | 003_saveload-set-b.c6h6_c6h6_t | 22 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -2.717000 |
| yes | 29 | 003_saveload-set-b.indole_c6h6_t | 23 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -5.627000 |
| yes | 30 | 003_saveload-set-b.phenol_phenol | 24 | 003_saveload-set-b | 003_saveload-set-b | ENERGY_DIFFERENCE | 3 | 1 | -7.097000 |

# Calculation completion for the current training set
# Reference: 25/30 (missing)
# Empty: 0/30 (missing)
# Additional (test3): 0/30 (missing)
# Terms: 
| H | l | 0.1 | 0/30 | (missing)
| H | l | 0.2 | 0/30 | (missing)
| H | l | 0.3 | 0/30 | (missing)
| C | l | 0.1 | 0/30 | (missing)
| C | l | 0.2 | 0/30 | (missing)
| C | l | 0.3 | 0/30 | (missing)
| N | l | 0.1 | 0/30 | (missing)
| N | l | 0.2 | 0/30 | (missing)
| N | l | 0.3 | 0/30 | (missing)
| N | s | 0.1 | 0/30 | (missing)
| N | s | 0.2 | 0/30 | (missing)
| N | s | 0.3 | 0/30 | (missing)
| O | l | 0.1 | 0/30 | (missing)
| O | l | 0.2 | 0/30 | (missing)
| O | l | 0.3 | 0/30 | (missing)
| O | s | 0.1 | 0/30 | (missing)
| O | s | 0.2 | 0/30 | (missing)
| O | s | 0.3 | 0/30 | (missing)
| O | p | 0.1 | 0/30 | (missing)
| O | p | 0.2 | 0/30 | (missing)
| O | p | 0.3 | 0/30 | (missing)
# Total terms: 0/630 (missing)

%% training delete bleh
%% training print
## Table of saved training sets in the database
| Name |

